health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/utils/utilities.js

```
@@ -212,27 +212,39 @@ const formValidator = (formData, key, state, t) => {
+    if (!Array.isArray(formData)) return null;
+  
+    // Regex: allow letters, numbers, and spaces only
     const noSpecialCharsExceptSpace = /^[A-Za-z0-9 ]+$/;
 
-    return noSpecialCharsExceptSpace.test(val);
-  }
-
-
+    const seen = new Set(); // To track unique non-empty values
+    // Loop through each value
+    for (let val of formData) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'val' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)